ystockquote.py

```diff
@@ -462,29 +468,67 @@ def get_short_ratio(symbol):
     return _request(symbol, 's7')
 
 
-def get_historical_prices(symbol, start_date, end_date):
+def _get_headers():
+    headers = {
+        'Connection': 'keep-alive',
+        'Expires': str(-1),
+        'Upgrade-Insecure-Requests': str(1),
+        # Google Chrome:
+        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) \
+                AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.99 Safari/537.36'  # noqa
+    }
+
+    return headers
+
+
+def _get_crumb_cookie(symbol):
+    # Scrape a history page for a valid crumb ID:
+    tu = "https://finance.yahoo.com/quote/{}/history".format(symbol)
+    req = Request(tu, headers=_get_headers())
+    resp = urlopen(req)
+    cookies = resp.info()['Set-Cookie']
+
+    content = str(resp.read().decode('utf-8').strip())
+
+    # Matches: {'crumb':'AlphaNumeric'}
+    rpat = '"CrumbStore":{"crumb":"([^"]+)"}'
+    crumb = re.findall(rpat, content)[0]
+    return {'crumb': crumb.encode('ascii').decode('unicode-escape'),
+            'cookie': cookies}
+
+
+def get_historical_prices(symbol, start_date, end_date, interval='1d'):
```

Comment:
Oh I see. Hmm... Either way seems fine. Maybe keep it because being able to request at a specific interval would be sweet :D although could you add a small docstring explaining the options?